components/datatypes/data/order.schema.json

```diff
@@ -52,6 +52,15 @@
           "description": "The total price of this order after all discounts and taxes have been applied.",
           "meta:titleId": "order##xdm:priceTotal##title##86151",
           "meta:descriptionId": "order##xdm:priceTotal##description##47781"
+        },
+        "xdm:orderType": {
+          "title": "Order Type",
+          "type": "string",
+          "description": "The type order that has been placed.",
+          "meta:enum": {
```

Comment:
i don’t think it’s an event type.We are doing this because we have two kinds of checkout, a regular checkout path and an instant purchase checkout path. This ‘property’ distinguishes between the two types of checkout events.

We will make the changes as suggested on enum.

Reply:
[mjasrotia](https://github.com/mjasrotia) I have opened up a new PR to address the comment-- Can you please follow up on this PR at https://github.com/adobe/xdm/pull/1537

Once the PR is merged -- I will close this PR

Reply:
Approved the other PR, please close this one